Runtime/Game/Requests/PlayerRequest.cs

```diff
@@ -357,35 +351,15 @@ namespace LootLocker
 {
     public partial class LootLockerAPIManager
     {
-        public static void LookupPlayerNames(LookupPlayerNamesRequest lookupPlayerNamesRequest, Action<PlayerNameLookupResponse> onComplete)
+        public static void LookupPlayerNames(string idType, string[] identifiers, Action<PlayerNameLookupResponse> onComplete)
```

Comment:
This change removes functionality that we had before.
With the previous function, you could look for several different ID's in a single request, this change swaps that out for just one ID per request.
It's a good function, but we should not remove the old function I think.

Reply:
Almost correct. It looks that way from this file. But it was only ever exposed in LootLockerSDKManager as f.ex. `LookupPlayerNamesBySteamIds`. So our users never "could" look up several different ID's in a single request. So instead of deprecating all those methods I elected to build on the current feature set and add more id types.